README.md

```diff
@@ -642,14 +688,19 @@ developer:test >
 ```
 
 As you can see the whole test flow was executed with a single `make` command. But the test was not successful - you can see that target that checks the diff between output and expected output failed. The reason for this is that you need to create
```

Comment:
> _...But the test was not successful - you can see that target.._
> _...you need to create. the expected file - you can do so by..._

I think is better use use commas rather than dashes (-)